### PR TITLE
Object explorer strict null checks

### PR DESCRIPTION
**Pull request by aaomidi**
https://github.com/microsoft/azuredatastudio/issues/9169

### CODE REVIEW

File: src/sql/platform/angularEventing/browser/angularEventingServiceImpl.ts [hunk 1]
```diff
@@ -24,8 +24,8 @@ export class AngularEventingService implements IAngularEventingService {
 		return emitter.event;
 	}
 
-	public sendAngularEvent(uri: string, event: AngularEventType, payload?: any): void {
-		const emitter = this._angularMap.get(uri);
+	public sendAngularEvent(uri: string | undefined, event: AngularEventType, payload?: any): void {
```
Comment (anthonydresser) [inline src/sql/platform/angularEventing/browser/angularEventingServiceImpl.ts:27]: was there a point in which we were passing in undefined here? I don't think undefined makes sense as an input here.

**Review by anthonydresser — CHANGES_REQUESTED** [review]
1 comment on a undefined use

File: src/sql/platform/angularEventing/browser/angularEventingServiceImpl.ts (hunk 1 above)
Comment (aaomidi) [reply src/sql/platform/angularEventing/browser/angularEventingServiceImpl.ts:27]: Didn't really think if it makes sense or not, i'll change it to require string

File: src/sql/platform/angularEventing/browser/angularEventingService.ts [hunk 2]
```diff
@@ -41,5 +41,5 @@ export interface IAngularEventingService {
 	 * @param uri Uri of the dashboard to send the event to
 	 * @param event event to send
 	 */
-	sendAngularEvent(uri: string, event: AngularEventType, payload?: any): void;
+	sendAngularEvent(uri: string | undefined, event: AngularEventType, payload?: any): void;
```
Comment (anthonydresser) [inline src/sql/platform/angularEventing/browser/angularEventingService.ts:44]: looks like you missed updating this. I assume this would throw an error.

**Pull request closed without merging**